temp-legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/components/CircleButtons.kt

```diff
@@ -72,9 +72,9 @@ fun CircleButtonFilled(
     Icon(
         modifier = modifier
             .clip(CircleShape)
-            .clickable(onClick = onClick)
             .background(backgroundColor, CircleShape)
-            .padding(clickAreaPadding), // enlarge click area
+            .padding(clickAreaPadding)
+            .clickable(onClick = onClick), // enlarge click area
```

Comment:
Clickable must come before padding; Otherwise the padding won't enlarge the click area

Reply:
thank you. I thought clickable area will include padding as well :(
Added previews as well